specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/namespaces-preview.json

```diff
@@ -0,0 +1,1339 @@
+            "$ref": "./examples/NameSpaces/NetworkSecurityPerimeterConfigurationReconcile.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/NamespaceNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceAssociationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Association reconciled"
+          },
+          "201": {
+            "description": "Request to reconcile the association accepted"
+          },
+          "202": {
+            "description": "Request to reconcile the association accepted."
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "EHNamespaceListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EHNamespace"
+          },
+          "description": "Result of the List Namespace operation"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of namespaces."
+        }
+      },
+      "description": "The response of the List Namespace operation"
+    },
+    "EHNamespace": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Properties of sku resource"
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "Properties of BYOK Identity description"
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../common/v1/definitions.json#/definitions/systemData"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "properties": {
+            "minimumTlsVersion": {
+              "type": "string",
+              "description": "Minimum TLS version for the incoming connections accepted by the namespace. The Default value would be set to Tls12",
+              "enum": [
+                "Tls12",
+                "Tls11",
+                "Tls"
```

Comment:
>Tls [](http://example.com/codeflow?start=17&length=3)

Tls10?

Reply:
Yes. If we don't qualify a version, then by default it is TLS10. That's the reason we don't specify a version for that